awx/main/models/events.py

```diff
@@ -794,12 +803,16 @@ class Meta:
         editable=False,
         db_index=False,
     )
+    # We need to keep this as an FK in the model because AdHocCommand uses a ManyToMany field
+    #   to hosts through adhoc_events. But in https://github.com/ansible/awx/pull/8236/ we
```

Comment:
Do we really need to keep that relationship at all? I can't actually find any places where it is used, and if it were to be used, there would likely be an alternative.

Reply:
For clarification, you don't see the Hosts field on AdHocCommand being used?